Dockerfile

```diff
@@ -4,7 +4,13 @@ ARG IMG_TAG=latest
 FROM golang:1.21-alpine AS gaiad-builder
 WORKDIR /src/app/
 COPY go.mod go.sum* ./
-RUN go mod download
+RUN set -eux; \
+    export ARCH=$(uname -m); \
+    WASM_VERSION=$(go list -m all | grep github.com/CosmWasm/wasmvm | awk '{print $2}'); \
+    if [ ! -z "${WASM_VERSION}" ]; then \
+      wget -O /lib/libwasmvm_muslc.a https://github.com/CosmWasm/wasmvm/releases/download/${WASM_VERSION}/libwasmvm_muslc.${ARCH}.a; \
+    fi; \
```

Comment:
Good call out, I actually forgot to  do the static link build tag in the make file. So in the event this fails, it will fail building and give the correct output that it could not find libwasmvm. Is nice because it keeps bash to a minimum

Reply:
ref makefile e9058051f89db82238d6b2e17eccaa41e39adcb5